### ISSUE TITLE
Doublequotes escaping is broken

### ISSUE THREAD

**Issue by spacentropy** [microsoft/K2Bridge#26]
Hi. It seems that double-quotes escape in filters doesnt'work:
Repro:
Get field and value with double-quotes, click "+", receive error.
<img width="963" alt="Untitled" src="https://user-images.githubusercontent.com/464801/76792241-ac052480-67d3-11ea-87cf-8064ac76e8dc.png">
The error itself:

```
Error: Request to Elasticsearch failed: {"error":{"root_cause":[{"type":"SyntaxException","reason":"Syntax error: Query could not be parsed: I could not parse that, sorry.. Query: 'let fromUnixTimeMilli = (t:long) {datetime(1970 - 01 - 01) + t * 1millisec};\nlet _data = database(\"database\").Table | where (message == \"Health check \"LF\" completed after 35.422ms with status Unhealthy and '\"{\\\"Url\\\":\\\"http://url\\\"}\"'\") and (Timestamp >= fromUnixTimeMilli(1584385027085) and Timestamp <= fromUnixTimeMilli(1584385927085));\n(_data | summarize count() by Timestamp = bin(Timestamp, 30s)\n| order by Timestamp asc | as aggs);\n(_data | order by Timestamp desc | limit 500 | as hits)'","index_uuid":"index","index":"index"}],"type":"QueryException","reason":"Failed executing azure data explorer query","phase":"query"},"status":500}
```

**Comment by spacentropy** [comment]
Here's repro:

```kql
.create table Purchases ( str:string, x:int)

.ingest inline into table Purchases <|
Shoes,1000
Wide Shoes,50
"Coats, black",20
"Coats with ""quotes""",5
```

Add this table to kibana, then click on `+` on `Coats with "quotes"` row
<img width="864" alt="Untitled" src="https://user-images.githubusercontent.com/464801/76889133-1bdbe380-6896-11ea-8765-6e8334d7789c.png">

**Comment by AsafMah** [comment]
Fixed with #156
